test/marginal/resolver-paths.cpp

```diff
@@ -6,10 +6,10 @@ using namespace vs;
 
 int test_normalizer(const char* expected, const char* parent, const char* child, bool allow_exit, bool base_dir){
     auto tmp = resolve_path::normalizer(parent,child,allow_exit,base_dir);
-    std::cout<<"╭<parent> "<<parent<<"\n";
-    std::cout<<"├<child>  "<<child<<"\n";
-    std::cout<<"├>        "<<tmp.second<<"\n";
-    std::cout<<"╰>>       "<<expected<<"\n";
+    std::cerr<<"╭<parent> "<<parent<<"\n";
```

Comment:
I know we discussed this change. If I manage to fix the resolver, I'll revert these changes before merging.